src/Extensions/ServiceFabric.Services/ServiceFabricServicesEventSource.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics.Tracing;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.ServiceFabric.Services
+{
+	/// <summary>
+	/// Service Fabric event source
+	/// </summary>
+	[EventSource(Name = "Microsoft-OMEX-Logs-SfServicesInit")] //Breaking Change: Name Changed from "Microsoft-OMEX-Logs"
```

Comment:
>SfServicesInit [](start = 42, length = 14)

is it possible to expand to 

Microsoft-OMEX-Logs-ServiceFabricServicesInitialization

Reply:
Renamed, thank you